doc/source/api-reference/qibo.rst

```diff
@@ -2414,51 +2414,155 @@ Alternatively, a Clifford circuit can also be executed starting from the :class:
     :members:
     :member-order: bysource
 
-Gate set tomography
-"""""""""""""""""""
-
-Gate set tomography is used to obtain the noise profile of each operator.
-The noise profile can then be inverted and encoded in basis operations to be
-used in probabilistic error cancellation.
+.. _Gate Set Tomography:
 
-.. testsetup:: python
+Gate Set Tomography
+-------------------
 
-    from qibo.noise import NoiseModel, DepolarizingError
-    from qibo import Circuit
-    from qibo import gates
-    from qibo.tomography.gate_set_tomography import *
-    import numpy as np
+The gate set tomography (GST) follows the procedure outlined in
+https://arxiv.org/pdf/1712.09271.pdf, which follows closely to that of the
+Linear inversion GST given in https://arxiv.org/pdf/1509.02921.pdf. The outputs
+of the functions, when used correctly, will allow one to set up the proper
+variables necessary for probabilistic error cancellation to mitigate noise
+arising from noisy quantum hardware. The codes allow for calibrating an empty
+circuit, and for one and two qubits of gate set tomography. We give some
+examples to illustrate the usage of the functions for clarity.
```

Comment:
Thanks for the comments. I didn't realise I was writing for PEC because my end-goal is to do PEC with GST. Anyway, let me work on the description for GST.